content/javascript/concepts/map/terms/set/set.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.set()'
+Description: 'Stores or updates entries of key-value pairs in a Map object.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Map'
+  - 'Object'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Stores or updates entries of key-value pairs in a `Map` object.
+
+## Syntax
+
+The `.set()` method aceepts two parameters, the `key` and the `value`, and assigns the entry to a map:
+
+```js
+const map = new Map();
+
+map.set(key, value);
+```
+
+The `key` and `value` can either be an object or a variable of any data type.
+
+## Example
+
+Key-value pairs are stored in a `Map` object using the `.set()` method.
+
+```js
+const fruits = new Map();
+fruits.set('Apples', 5);
+fruits.set('Oranges', 8);
+```
+
+The example above uses `.set()` to store two entries in the `fruits` map: one for 5 apples and another for 8 oranges. These entries can be overwritten and have their values changed with `.set()` as well:
+
+```js
+fruits.set('Apples', 2);
+console.log(fruits);
+```
+
+The output will be:
+
+```
```

Comment:
```suggestion
```pseudo
```

Reply:
Not absolutely needed but just to preface the output anyways~

Reply:
Let's leave it as-is for now since this is more like shell-output. Totally open to changing later, though. 😃